scripts/service_install.sh

```diff
@@ -4,6 +4,7 @@ set -euxo pipefail
 service_name="gatestext.service"
 
 dir=$(pwd)
-eval "sed s#{WORKDIR}#$dir# $dir/services/gatestext.service > /lib/systemd/system/gatestext.service"
-chmod 755 /lib/systemd/system/$service_name
-systemctl daemon-reload
\ No newline at end of file
+cp $dir/bin/gatestext_script_linux /usr/local/bin
+cp $dir/services/gatestext.service $1
```

Comment:
Can you add a comment about what the expectation for the `$1` path is

Reply:
addressed